client/src/App.js

```diff
@@ -22,7 +23,20 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <div>
+        <Route 
+          exact path="/" 
+          component={MovieList} 
+        />
+        <Route 
+          render={(props) => 
+            <Movie 
+              addToSavedList={this.addToSavedList} 
+              {...props} 
+            />} 
+          path="/movies/:id" 
```

Comment:
This should be beside or below the Route - not after the render. It shouldn't cause bugs, but it's about creating continuity in your code.